Dockerfile

```diff
@@ -134,14 +136,22 @@ ENV LD_LIBRARY_PATH $LD_LIBRARY_PATH:/usr/local/cuda/extras/CUPTI/lib64:/usr/loc
 # << END Configure Bazel
 
 
-
-
 # Copy DeepSpeech repo contents to container's /DeepSpeech
 COPY . /DeepSpeech/
 
+# Alternative clone from GitHub 
+# RUN apt-get update && apt-get install -y \
+#    software-properties-common \
+#    curl
+# RUN add-apt-repository ppa:git-core/ppa && \
```

Comment:
You really don't need those two lines, just `apt-get install git-lfs`, it's packaged in Ubuntu 18.04, so no reason it should not work.

Reply:
fixed

Reply:
thanks!